tsplib95/models.py

```diff
@@ -144,7 +144,7 @@ def is_complete(self):
 
         :rtype: bool
         """
-        return 'EDGE_DATA_FORMAT' not in self
+        return (self.edge_data_format is None)
```

Comment:
Let's instead use `return not bool(self.edge_data_format)` to account for empty but existing values as well. Same for the other's down below: drop the parens and use the inherent truth of the field.

Reply:
Good point, I changed my pull request to do it the way you proposed.